Test/QuickCheck/Gen.hs

```diff
@@ -197,7 +198,7 @@ frequency xs0 = choose (1, tot) >>= (`pick` xs0)
   pick _ _  = error "QuickCheck.pick used with empty list"
 
 -- | Generates one of the given values. The input list must be non-empty.
-elements :: [a] -> Gen a
+elements :: HasCallStack => [a] -> Gen a
```

Comment:
This only works with recent versions of GHC.

My `call-stack` package provides a shim for backwards compatibility.  @nick8325 are you fine with adding that dependency?  If yes, we could also add source locations to e.g.  `===`.

If not, we would want to CPP the `HasCallStack => ` of the type signature.

Reply:
👍